ci/Jenkinsfile-kubernetes

```diff
@@ -361,9 +419,13 @@ node("docker") {
     }
 
     parallel(getBuildAndPublishStages(props))
+*/
 
     getKubernetesSparkVersions(props).each { sparkMajorVersion ->
         stage("Test Scala internal backend, Spark ${sparkMajorVersion}") {
+            def valuesAsString = "version=3.32.0.1-1\ndocker_registry_id=524466471676\ncluster_name=SWKubernetes-8ZRRJyAG\ncluster_endpoint=https://91F60C742ECD736D3441B2D8FA007C6E.gr7.us-east-2.eks.amazonaws.com\n"
```

Comment:
Same here, for debugging purposes only, refer to existing cluster -> will remove before merging

Reply:
Removed